scripts/fixtures.sql

```diff
@@ -1,14 +1,23 @@
-create procedure department(name text)
+create procedure department(lectures_root realms.id%type, name text)
```

Comment:
I think I would reverse the order of all of these `procedure` declarations. :thinking: